PathToRegex/PathToRegex.swift

```diff
@@ -20,7 +20,12 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
-import Regex
+
+#if COCOAPODS
+    import CrossroadRegex
```

Comment:
No pb thks for all the hard work!

Sent from my iPhone


> On Jun 18, 2016, at 12:36 PM, Daniel Leping <notifications@github.com> wrote:
> 
> In PathToRegex/PathToRegex.swift:
> 
> > @@ -20,7 +20,12 @@
> >  //===----------------------------------------------------------------------===//
> >  
> >  import Foundation
> > -import Regex
> > +
> > +#if COCOAPODS
> > +    import CrossroadRegex
> Hey @pgherveou , I have finally managed to go through Regex framework and apply the changes. Still, there are major breaking changes (Xcode 8... you know) in release 0.7, so I will need some time to adjust PathToRegex as well.
> 
> Thanks for your patience!
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>